app/models/profile.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 # OpenSCAP profile
-class Profile < ApplicationRecord
+class Profile < ApplicationRecord # rubocop:disable Metrics/ClassLength
```

Comment:
idk, this model doesn't feel too big to me. I'd be willing to bump up the max class length from 100 lines to 150.

Reply:
I'd say it'd be best not to start with that - maybe in this class it's OK though since the `result` method is quite a long SQL string